private/inc/avb_watchdog/IasSystemdWatchdogManager.hpp

```diff
@@ -0,0 +1,186 @@
+/*
+ @COPYRIGHT_TAG@
```

Comment:
We are using a proper License text right now, I think this should be fixed before merging:
(Even, I think, if you aren't the original author)

```
/*
 * Copyright (C) 2018 Intel Corporation. All rights reserved.
 *
 * SPDX-License-Identifier: BSD-3-Clause
 */
```

Reply:
True, I'll fix that up. Thanks !

Reply:
Done.